index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile First Project</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="logo">My Logo</a>
+      <nav class="hamburger">
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+      </nav>
+    </header>
+
+    <section class="main-container">
+      <div class="title">
+        <h1>This is Mert Glad to see you here!</h1>
+      </div>
+      <dev class="intro">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+        <p class="connect"><a href="#">LET'S CONNECT</a></p>
+
+        <div class="social-media">
+          <ul class="social-media-icons">
+            <li>
+              <a href="https://twitter.com/aranelluinwe">
+                <img class="icons" src="Images/twitter.png" alt="twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/mert-kantaroglu/">
+                <img
+                  class="icons"
+                  src="Images/Linkedin.png"
+                  alt="linkedin logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://medium.com/@mertkantaroglu">
+                <img class="icons" src="Images/medium.png" alt="medium logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://github.com/mertkantaroglu">
+                <img class="icons" src="Images/github.png" alt="github logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  class="icons"
+                  src="Images/angellist.png"
+                  alt="angellist logo"
+              /></a>
+            </li>
+          </ul>
```

Comment:
- [ ] Dear @mertkantaroglu, I see you have a problem removing the list items from your previous comment. please use `li {list-style: none;}` to make listed without bullet. 

![Capture](https://user-images.githubusercontent.com/69077061/205226296-1d94edb1-ae8d-4d4f-b876-e5023f467621.PNG)

Reply:
Thanks Tadesse, it worked.

Reply:
Good job @mertkantaroglu ✋